op-program/host/cmd/main_test.go

```diff
@@ -54,181 +55,196 @@ func TestDefaultCLIOptionsMatchDefaultConfig(t *testing.T) {
 
 func TestNetwork(t *testing.T) {
 	t.Run("Unknown", func(t *testing.T) {
-		verifyArgsInvalid(t, "invalid network bar", replaceRequiredArg(t, "--network", "bar"))
+		verifyArgsInvalid(t, "invalid network bar", replaceRequiredArg("--network", "bar"))
 	})
 
 	t.Run("Required", func(t *testing.T) {
-		verifyArgsInvalid(t, "flag rollup.config or network is required", addRequiredArgsExcept(t, "--network"))
+		verifyArgsInvalid(t, "flag rollup.config or network is required", addRequiredArgsExcept("--network"))
 	})
 
 	t.Run("DisallowNetworkAndRollupConfig", func(t *testing.T) {
-		verifyArgsInvalid(t, "cannot specify both rollup.config and network", addRequiredArgs(t, "--rollup.config=foo"))
+		verifyArgsInvalid(t, "cannot specify both rollup.config and network", addRequiredArgs("--rollup.config=foo"))
 	})
 
 	t.Run("RollupConfig", func(t *testing.T) {
-		dir := t.TempDir()
-		configJson, err := json.Marshal(chaincfg.Goerli)
-		require.NoError(t, err)
-		configFile := dir + "/config.json"
-		err = os.WriteFile(configFile, configJson, os.ModePerm)
-		require.NoError(t, err)
-
-		cfg := configForArgs(t, addRequiredArgsExcept(t, "--network", "--rollup.config", configFile))
+		configFile := writeValidRollupConfig(t)
+		genesisFile := writeValidGenesis(t)
+
+		cfg := configForArgs(t, addRequiredArgsExcept("--network", "--rollup.config", configFile, "--l2.genesis", genesisFile))
 		require.Equal(t, chaincfg.Goerli, *cfg.Rollup)
 	})
 
 	for name, cfg := range chaincfg.NetworksByName {
 		name := name
 		expected := cfg
 		t.Run("Network_"+name, func(t *testing.T) {
-			cfg := configForArgs(t, replaceRequiredArg(t, "--network", name))
+			args := replaceRequiredArg("--network", name)
+			if name == "beta-1" {
+				// No built-in config for beta-1 which is fine as it's no longer in use.
+				// Newly added named networks should hook up the L2 chain config
```

Comment:
nit: Maybe we should remove `beta-1` from the chaincfg networks if it's no longer used.

Reply:
Tracking in https://linear.app/optimism/issue/CLI-3887/remove-beta-1-network-definition